deploy/frontend.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: provisioning-frontend
+objects:
+  - apiVersion: cloud.redhat.com/v1alpha1
+    kind: Frontend
+    metadata:
+      name: provisioning
```

Comment:
I think this has to match `provisioning-frontend` resourceTemplate name for `deploy-iqe-cji` to correctly identify the component.

Reply:
ok, lets try with provisioning-frontend, but this is what I've copied from https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/deploy/frontend.yaml